### ISSUE TITLE
[Infinite Loop] JS Slot Machine

### ISSUE THREAD

**Issue by patrickjtoy** [FreeCodeCamp/FreeCodeCamp#2527]
Challenge http://freecodecamp.com/challenges/waypoint-bring-your-javascript-slot-machine-to-life has an issue. Please describe how to reproduce it, and include links to screenshots if possible.

1. In the space provided for code, type `$($)`
2. This causes an infinite loop.

![image](https://cloud.githubusercontent.com/assets/5121391/9425943/16780828-48f3-11e5-978e-49c241e6ab7a.png)

**Comment by ahstro** [comment]
This is a duplicate of #2286, please close.

**Comment by patrickjtoy** [comment]
Duplicate of #2516 - I was able to workaround by removing the key from local storage.

**Comment by arxene** [comment]
@patrickjtoy Can you explain in further detail your solution? This would be useful as this bug has been opened by several people.

**Comment by arxene** [comment]
This MDN guide to [DOM storage](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Storage#Storage_location_and_clearing_the_data) mentioned clearing cookies. I'm using Chrome rather than Firefox, but i tried clearing my browser history which included clearing cookies and I was then able to refresh Basic JS Waypoint 39 with the old code I'd entered cleared.

**Comment by patrickjtoy** [comment]
@arxene In Chrome, you would open the dev tools (F12), click "Resources", expand "Local Storage" in the left panel, click the URL for FCC, find the key for the exercise that's stuck, right-click that key and choose "Delete"

**Comment by cedevita** [comment]
I had the same problem.. pressed Ctrl+enter by mistake - ended up bugging Chrome tab. Couldn't use "Learn" option on FreeCodeCamp, since it would always take me to the "bugged" page it automatically saved.

My solution:
I copied link from bugged page, pasted in a new tab, and made change after "?solution="... Actually I deleted almost everything, and my code of course did not work, but most important - chrome was not bugged.  
After that, i just used "reset" code option..
Good luck!
